R/indentation_linter.R

```diff
@@ -261,6 +297,16 @@ indentation_linter <- function(indent = 2L, hanging_indent_style = c("tidy", "al
   })
 }
 
+find_new_indent <- function(current_indent, change_type, indent, hanging_indent) {
+  if (change_type == "hanging") {
```

Comment:
nit: maybe a `switch()` is more appropriate? `change_type` feels pretty `enum`-like